core/src/main/java/org/apache/calcite/sql/fun/SqlJsonQueryFunction.java

```diff
@@ -45,12 +45,24 @@
 import java.util.List;
 import java.util.Optional;
 
+import static org.apache.calcite.sql.type.OperandTypes.family;
+
 import static java.util.Objects.requireNonNull;
 
 /**
  * The <code>JSON_QUERY</code> function.
  */
 public class SqlJsonQueryFunction extends SqlFunction {
+  static final SqlSingleOperandTypeChecker TYPE_CHECKER =
```

Comment:
I have a small question, can it be written like this here?
```
  public static final SqlSingleOperandTypeChecker TYPE_CHECKER =
      family(SqlTypeFamily.ANY, SqlTypeFamily.CHARACTER,
          SqlTypeFamily.ANY, SqlTypeFamily.ANY, SqlTypeFamily.ANY)
          .or(family(SqlTypeFamily.ANY));
```

Reply:
I think that the form you propose means "either 5 arguments, or just 1".
ANY means "the argument may have any type".
But what we need is "5 or 6".

Reply:
Thanks! Got it.